tests/Unit/Domain/FunctionsOfTime/Test_SettleToConstantQuaternion.cpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <array>
+#include <memory>
+
+#include "DataStructures/DataVector.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "Domain/FunctionsOfTime/RegisterDerivedWithCharm.hpp"
+#include "Domain/FunctionsOfTime/SettleToConstantQuaternion.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+
+namespace {
+void test(
+    const std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>& f_of_t,
+    const double match_time, const double decay_time, const DataVector& f_t0,
+    const DataVector& dtf_t0, const DataVector& d2tf_t0, const DataVector& A) {
+  // check that values agree at the matching time
+  const auto lambdas0 = f_of_t->func_and_2_derivs(match_time);
+  CHECK_ITERABLE_APPROX(lambdas0[0], f_t0);
+  CHECK_ITERABLE_APPROX(lambdas0[1], dtf_t0);
+  CHECK_ITERABLE_APPROX(lambdas0[2], d2tf_t0);
+
+  const auto lambdas1 = f_of_t->func_and_deriv(match_time);
+  CHECK_ITERABLE_APPROX(lambdas1[0], f_t0);
+  CHECK_ITERABLE_APPROX(lambdas1[1], dtf_t0);
+
+  const auto lambdas2 = f_of_t->func(match_time);
+  CHECK_ITERABLE_APPROX(lambdas2[0], f_t0);
+
+  // check that asymptotic values approach f = A/|A| = const.
+  const auto lambdas3 = f_of_t->func_and_2_derivs(1.0e5);
+  const DataVector zero{0.0, 0.0, 0.0, 0.0};
+  const DataVector normalized_A = A / sqrt(sum(A * A));
+  CHECK_ITERABLE_APPROX(lambdas3[0], normalized_A);
+  CHECK_ITERABLE_APPROX(lambdas3[1], zero);
+  CHECK_ITERABLE_APPROX(lambdas3[2], zero);
+
+  const auto lambdas4 = f_of_t->func_and_deriv(1.0e5);
+  CHECK_ITERABLE_APPROX(lambdas4[0], normalized_A);
+  CHECK_ITERABLE_APPROX(lambdas4[1], zero);
+
+  const auto lambdas5 = f_of_t->func(1.0e5);
+  CHECK_ITERABLE_APPROX(lambdas5[0], normalized_A);
+
+  // Check that function of time remains a unit quaternion
+  const auto lambdas6 = f_of_t->func(match_time + decay_time * 0.34);
+  const double lambdas6_norm = sqrt(sum(square(lambdas6[0])));
+  CHECK(approx(lambdas6_norm) == 1.0);
+
+  // test time_bounds function
+  const auto t_bounds = f_of_t->time_bounds();
+  CHECK(t_bounds[0] == match_time);
+  CHECK(t_bounds[1] == std::numeric_limits<double>::infinity());
+  CHECK(f_of_t->expiration_after(match_time) ==
+        std::numeric_limits<double>::infinity());
+}
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Domain.FunctionsOfTime.SettleToConstantQuaternion",
+                  "[Domain][Unit]") {
+  using SettleToConstantQuaternion =
+      domain::FunctionsOfTime::SettleToConstantQuaternion;
+
+  domain::FunctionsOfTime::register_derived_with_charm();
+
+  const double match_time = 10.0;
+  const double decay_time = 5.0;
+  // Choose for an initial quaternion
+  // q = (cos \theta/2, \hat{n} \sin \theta/2), where
+  // \theta = \Omega(t-t_0) + \alpha(t-t_0)^2/2 _ \phi
```

Comment:
Is the underscore supposed to be a minus?

Reply:
should be a +...thanks for catching this!